assessment.sql

```diff
@@ -5,19 +5,38 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees (id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  height CHARACTER VARYING,
+  weight CHARACTER VARYING,
+  dob CHARACTER VARYING,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+  );
+
+Created Database, Created Table, Bulk Loaded Data
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees (surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
+
+It worked!
+SELECT * FROM licensees;
+...
+16 | Bishop    | Caren      | f      | 63     | 132    | 1943-09-26 | Brown     | Black
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+Opps! I did this step before I added Caren
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM './data/licensees.csv' WITH (FORMAT csv, HEADER true)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+It is telling me my syntax is wrong
+SELECT * FROM licensees WHERE eye_color = Hazel WHERE hair_color = Brown WHERE hair_color = Black;
```

Comment:
You get one WHERE, and your entire condition follows. If your condition is more complex than checking the value of one column, then you need to use logical operators to combine them. Here is an example of a WHERE clause that would work for this query:

```sql
...
WHERE eye_color = 'Hazel' AND (hair_color = 'Black' OR hair_color = 'Brown')
...
```

Note, also, we compare the values in the columns to strings; if you don't use quotes, you're comparing them to values in other columns -- columns that don't exist, in this case. 2/5